### DISCUSSION TITLE
Cinnamon menu: frecency and fuzzy finding

### DISCUSSION THREAD

**Discussion by returnDanilo** [linuxmint/discussions#720] [Ideas]
This would be a quality of life improvement for launching apps:

* The fuzzy finding would allow users to not need to spell words completely correctly to see corresponding apps listed.
* The frecency would allow users to press return right after writing a search query without the need to always use the mouse/arrow keys to select the desired app.

**Comment by Jopp-gh** [comment]
I guess you're talking about menu@cinnamon.org.

If so, cinnamon menu is a good applet, but I prefer app launchers like Ulauncher, because visually minimal and with fuzzy search.
